packages/frontend/core/src/desktop/pages/auth/oauth-callback.tsx

```diff
@@ -80,8 +81,17 @@ export const Component = () => {
         // TODO(@forehalo): need a good way to go back to previous tab and close current one
         nav(redirectUri ?? '/');
       })
-      .catch(e => {
-        nav(`/sign-in?error=${encodeURIComponent(e.message)}`);
+      .catch(err => {
+        const userFriendlyError = UserFriendlyError.fromAnyError(err);
```

Comment:
Because this route is only valid for the web, the web is unlikely to trigger the minimum version restriction, so this may be useless